src/clauses/Clause.ts

```diff
@@ -27,16 +27,23 @@ import { toCypherParams } from "../utils/to-cypher-params";
 
 const customInspectSymbol = Symbol.for("nodejs.util.inspect.custom");
 
+/** Config fields for the .build method */
+export type BuildConfig = Partial<EnvConfig>;
+
 /** Represents a clause AST node
  *  @group Internal
  */
 export abstract class Clause extends CypherASTNode {
     protected nextClause: Clause | undefined;
 
     /** Compiles a clause into Cypher and params */
-    public build(prefix?: string | EnvPrefix | undefined, extraParams: Record<string, unknown> = {}): CypherResult {
+    public build(
+        prefix?: string | EnvPrefix | undefined,
+        extraParams: Record<string, unknown> = {},
+        config?: BuildConfig
+    ): CypherResult {
```

Comment:
No action required, just a thought for the future: If the function params were instead an object, then your example would be the following, which is a little bit nicer (no need to pass in your unused params). 

```
const queryResult = new TestClause(query).build({
    config: {
        labelOperator: "&",
    }
)};
```